samples/javascript/host.json

```diff
@@ -0,0 +1,13 @@
+{
+    "http": {
+      "routePrefix": ""
+    },
+    "applicationInsights": {
+      "sampling": {
+        "isEnabled": false
+      }
+    },
+    "durableTask": {
+      "HubName": "SampleHubCsx"
```

Comment:
Let's name the hub something else since this name implies C# scripting (csx).

Reply:
Good catch, thanks.